app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def calculate_team_total_rating(player_list: list) -> int:
+    return sum([player.get_rating() for player in player_list])
```

Comment:
please, read the checklist: https://github.com/mate-academy/py-elves-and-dwarves/blob/master/checklist.md#code-efficiency